test/integration/targets/vmware_content_library/content_lib_crud_ops.yml

```diff
@@ -0,0 +1,73 @@
+# Test code for the vmware_content_library CRUD Operations.
```

Comment:
Thank you for the functional test, could you move the file here: test/integration/targets/vmware_content_library/tasks/main.yml

Reply:
Actually , I will be adding more modules for content library functionalities. Thought of having all test .ymls in single directory. if you insist lets have 1:1 mapping then I will make this change?

Reply:
Actually , I will be adding more modules for content library functionalities. Thought of having all test .ymls in single directory. if you insist lets have 1:1 mapping then I will make this change?

Reply:
The CI needs the mapping to trigger to correct test role depending on the module that has been changed. So yes, please, break them up. If you want to share some common tasks between several roles, you can add a new playbook in `prepare_vmware_tests`. https://github.com/ansible/ansible/tree/devel/test/integration/targets/prepare_vmware_tests/tasks